docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/global-permissions.md

```diff
@@ -249,3 +286,82 @@ To refresh group memberships,
 1. Click **Refresh Group Memberships**.
 
 **Result:** Any changes to the group members' permissions will take effect.
+
+## Restricted Admin
+
+:::warning Deprecated
+
+The Restricted Admin role is deprecated, and will be removed in a future version of rancher (2.10 or higher). Users are encouraged to make a custom role with the desired permissions rather than relying on this built-in role).
+
+:::
+
+A new `restricted-admin` role was created in Rancher v2.5 in order to prevent privilege escalation from the local Rancher server Kubernetes cluster. This role has full administrator access to all downstream clusters managed by Rancher, but it does not have permission to alter the local Kubernetes cluster.
+
+The `restricted-admin` can create other `restricted-admin` users with an equal level of access.
+
+A new setting was added to Rancher to set the initial bootstrapped administrator to have the `restricted-admin` role. This applies to the first user created when the Rancher server is started for the first time. If the environment variable is set, then no global administrator would be created, and it would be impossible to create the global administrator through Rancher.
```

Comment:
```suggestion
The initial bootstrapped administrator account has the `restricted-admin` role. The bootstrapped admin is the first user created when the Rancher server is initialized. If the environment variable is set, then no global administrator then no global administrator is created, and you are prevented from creating the global administrator through Rancher
```

<s>"If the environment variable is set" -- meaning `CATTLE_RESTRICTED_DEFAULT_ADMIN=false`? We should specify that here.</s>

After re-reading a couple of times -- are we saying that 2.8 introduces `CATTLE_RESTRICTED_DEFAULT_ADMIN`?

In that case:

```suggestion
Rancher v2.8.0 introduces `CATTLE_RESTRICTED_DEFAULT_ADMIN`, an environment variable which ensures that the initial bootstrapped administrator account has the `restricted-admin` role. The bootstrapped admin is the first user created when the Rancher server is initialized. If the environment variable is set to `true`, then no global administrator is created, and you are prevented from creating the global administrator through Rancher.
```